webapp/app.py

```diff
@@ -407,6 +408,12 @@ def utility_processor():
     methods=["PUT"],
 )
 
+app.add_url_rule(
+    "/advantage/offers",
```

Comment:
The endpoint path conflicts with the one Clem needs for the HTML template. No?

We do not have a convention for our endpoints to avoid these conflicts. Any suggestions are welcome:
- we did `.json` suffix at some point
- perhaps doing `/api/v1` might become handy too

Would be worth discussing.